packages/react-native-bridge/index.js

```diff
@@ -175,6 +179,10 @@ export function mediaUploadSync() {
 	return RNReactNativeGutenbergBridge.mediaUploadSync();
 }
 
+export function mediaSaveSync() {
+	return RNReactNativeGutenbergBridge.mediaSaveSync();
+}
```

Comment:
Could we add JDocs descriptions to this function too? 🙏

Reply:
👍 good idea, added in 08e77ba07